src/main.js

```diff
@@ -232,7 +232,10 @@ $(document).ready(() => {
         isMouseInSidebar = false;
         startTimer(MOUSE_LEAVE_DELAY);
       });
-      $sidebar.on('mouseenter mousemove', () => {
+      $sidebar.on('mouseenter mousemove', (event) => {
+        // Mouse is out
```

Comment:
Can you add more detailed comment here and why does this fixes it. I'm sure 3 months later when we look back, we'll be surprised what this is about.